src/discussions/topics/topic-group/topic/Topic.jsx

```diff
@@ -1,58 +1,69 @@
+/* eslint-disable no-unused-vars, react/forbid-prop-types */
 import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons';
 import { faComments, faFlag } from '@fortawesome/free-solid-svg-icons';
-
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { useParams } from 'react-router';
 import { Link } from 'react-router-dom';
 import { Routes } from '../../../../data/constants';
 
-// eslint-disable-next-line no-unused-vars
-function Topic({ id, name, topics }) {
+function Topic({
+  id,
+  name,
+  questions,
+  discussions,
+  flags,
+}) {
   const { courseId } = useParams();
 
   return (
-    <div className="discussion-topic d-flex flex-column border-bottom pl-2 pt-1 pb-1" data-topic-id={id}>
-      <Link
-        className="topic-name"
-        to={
-          Routes.POSTS.PATH.replace(':courseId', courseId)
-            .replace(':topicId', id)
-            .replace(':threadId', '')
-        }
-      >
-        { name }
-      </Link>
-      <div className="d-flex">
-        <span className="badge mr-1">
-          <FontAwesomeIcon icon={faQuestionCircle} />
-          22
-        </span>
-        <span className="badge mr-1">
-          <FontAwesomeIcon icon={faComments} />
-          33
-        </span>
-        <span className="badge">
-          <FontAwesomeIcon icon={faFlag} />
-          5
-        </span>
+    <Link
+      className="discussion-topic d-flex flex-column list-group-item px-3 py-2 text-gray-900 text-decoration-none"
+      data-topic-id={id}
+      to={
+        Routes.POSTS.PATH.replace(':courseId', courseId)
+          .replace(':topicId', id)
+      }
+    >
+      <div className="topic-name font-weight-bold small">
+        {name}
+      </div>
+      <div className="d-flex text-gray-300 h4 mt-1 mb-0">
+        <div className="badge mr-4">
+          <FontAwesomeIcon className="mr-2" icon={faQuestionCircle} />
+          {questions}
+        </div>
+        <div className="badge mr-4">
+          <FontAwesomeIcon className="mr-2" icon={faComments} />
+          {discussions}
+        </div>
+        {flags!==null && (
+          <div className="badge">
+            <FontAwesomeIcon className="mr-2" icon={faFlag} />
+            {flags}
```

Comment:
use paragon icons

Reply:
I think we do not (yet) have these icons in Paragon, so what we can do is to have a ticket in the backlog to replace those with paragon icons and remove the dependency of FA. This shouldn't be a blocker.

Reply:
I think Icons are now available but we decided to refactor icons in a separate PR. hence marking it resolve